api/kv/value/value.go

```diff
@@ -130,6 +135,32 @@ func Uint(v uint) Value {
 	return Uint64(uint64(v))
 }
 
+// Array stores array values as an attribute
+func Array(array interface{}) Value {
+	switch reflect.TypeOf(array).Kind() {
+	case reflect.Array, reflect.Slice:
+		isValidType := func() bool {
+			// get array type regardless of dimensions
+			typeName := reflect.TypeOf(array).String()
+			typeName = typeName[strings.LastIndex(typeName, "]")+1:]
```

Comment:
No need to parse, we can just use the `Elem` method.

```suggestion
			typeName := reflect.TypeOf(array).Elem().String()
```

https://golang.org/pkg/reflect/#Type

Reply:
The reason, I choose to parse string is, in case of multidimensional array e.g `[][]bool` `Elem()` will return type `[]bool` so thought parse string and get type once.